build/scripts/build.ps1

```diff
@@ -180,22 +186,17 @@ function Run-MSBuild([string]$projectFilePath, [string]$buildArgs = "", [string]
 # Restore all of the projects that the repo consumes
 function Restore-Packages() {
     Write-Host "Restore using dotnet at $dotnet"
-
-    $all = @(
-        "Roslyn Toolset:build\ToolsetPackages\RoslynToolset.csproj",
-        "Roslyn:Roslyn.sln")
-
-    foreach ($cur in $all) {
-        $both = $cur.Split(':')
-        Write-Host "Restoring $($both[0])"
-        $projectFilePath = $both[1]
-        $projectFileName = [IO.Path]::GetFileNameWithoutExtension($projectFilePath)
-        $logFilePath = ""
-        if ($binaryLog) {
-            $logFilePath = Join-Path $logsDir "Restore-$($projectFileName).binlog"
-        }
-        Restore-Project $dotnet $both[1] $logFilePath
-    }
+    
+    Write-Host "Restoring Roslyn Toolset"
+    $logFilePath = if ($binaryLog) { Join-Path $logsDir "Restore-RoslynToolset.binlog" } else { "" }    
+    Restore-Project $dotnet "build\ToolsetPackages\RoslynToolset.csproj" $logFilePath
+
+    $logFilePath = if ($binaryLog) { Join-Path $logsDir "RepoToolsetBuild.binlog" } else { "" }        
+    Restore-Project "build\Targets\RepoToolset\Build.proj" "/p:Restore=true /bl:$logFilePath" -summary:$false
```

Comment:
>-summary:$false [](start = 94, length = 15)

There is no `-summary` parameter in Restore-Project.

Reply:
Sorry, I was confused. It's been a while since I wrote this. The previous version was correct - I didn't call `Restore-Project` here because it's not actually invoking `restore` target but building with `/p:Restore=true`.

Reply:
Why are you building during restore?

Reply:
`Build.proj` does restore.

Reply:
This is the full Build.proj from RepoToolset: https://github.com/dotnet/roslyn-tools/blob/master/sdks/RepoToolset/tools/Build.proj
For now I'm just using part of it.